src/tests/ftest/scrubber/target_auto_eviction.py

```diff
@@ -32,10 +34,31 @@ def test_scrubber_ssd_auto_eviction(self):
         self.create_pool_cont_with_scrubber(pool_prop=pool_prop, cont_prop=cont_prop)
         self.dmg_cmd.pool_query(self.pool.identifier)
         initial_metrics = self.scrubber.get_scrub_corrupt_metrics()
+        t_start = journalctl_time()
+        t_end = journalctl_time(datetime.now() + timedelta(seconds=60))
         self.run_ior_and_check_scruber_status(pool=self.pool, cont=self.container)
         # Wait for a minute for the scrubber to take action and evict target
         # after corruption threshold reached.
+        self.log.info("Sleeping for 60 seconds")
         time.sleep(60)
+        # Check the journalctl for data corrupt message.
+        command = get_journalctl_command(since=t_start, until=t_end, system=True,
+                                         units="daos_server")
+        err = "Error gathering system log events"
+        results = get_host_data(hosts=self.hostlist_servers, command=command, text="journalctl",
+                                error=err)
+        self.log.info(results)
+        str_to_match = "Data corruption detected"
+        occurrence = 0
+        for count, _ in enumerate(self.hostlist_servers):
+            occurrence = results[count]["data"].count(str_to_match)
+            if occurrence > 0:
+                break
```

Comment:
Minor - We could just directly loop over the results
```suggestion
        for host_result in results:
            occurrence = host_result["data"].count(str_to_match)
            if occurrence > 0:
                break
```

Reply:
Done.